app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -65,6 +65,11 @@ public function store(ReportbackRequest $request)
      */
     public function updateReportbackItems(Request $request)
     {
+        $this->validate($request, [
+            '*.rogue_reportback_item_id' => 'required',
```

Comment:
@sbsmith86 since the request is an array of reportback items, this is something new in Laravel 5.2 to iterate through and make sure each reportback item has `rogue_reportback_item_id` and `status`. The `*` indicates the index.